src/cmd/builtin/pty.c

```diff
@@ -1,977 +0,0 @@
-/***********************************************************************
```

Comment:
What do you plan to do with `pty` ? Currently it's being built as a standalone command. If we want to retain it, it does not make sense to remove this code.

Reply:
I had a second commit turning it into a builtin that I forgot to push. With a couple of small changes to block SIGCHLD so that the normal ksh job termination code didn't reap pty's child process it seemed to be working on macOS. Then this morning I tested it on OpenSuse and it fails in `_sffilbuf()` (from the `sfgetr()` call in *pty.c*). Which suggests there may be bugs in the code (quell surprise). So for the moment I'm going to reinstate it as a standalone command.

Perhaps we should reopen the question of whether we want to keep `pty` as the only other external command we build.

Reply:
+1 for reopening the discussion to keep `pty` command.